src/conv/invokers/mlir_impl_gemm.cpp

```diff
@@ -77,15 +103,32 @@ void ComputeMlirDimsStrides(const conv::ProblemDescription& conv_problem,
                             std::vector<size_t>& out_dims,
                             std::vector<size_t>& out_strides)
 {
+    auto group_count = conv_problem.GetGroupCount();
+
+    // Add a virtual group dimenions before input channel
     const TensorDescriptor& in = conv_problem.GetIn();
-    std::make_tuple(in_dims, in_strides) = PermuteDimStride(in.GetLengths(), in.GetStrides());
+    in_dims                    = in.GetLengths();
+    in_strides                 = in.GetStrides();
+    std::make_tuple(in_dims, in_strides) =
+        InsertGToDimStride(in.GetLayout("NCHW"), 'C', in_dims, in_strides, group_count);
```

Comment:
[Comment] First two lines are not actually needed
```suggestion
    std::make_tuple(in_dims, in_strides) =
        InsertGToDimStride(in.GetLayout("NCHW"), 'C', in.GetLengths(), in.GetStrides(), group_count);
```

Reply:
Will update once I make dims and strides both input/output.

Reply:
That [comment] is just FYI, no response needed ;)